packages/page-staking/src/Overview/SummaryNominators.tsx

```diff
@@ -0,0 +1,93 @@
+// Copyright 2017-2022 @polkadot/app-staking authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+
+import { Card, CardSummary, SummaryBox } from '@polkadot/react-components';
+import { FormatBalance } from '@polkadot/react-query';
+import { formatNumber } from '@polkadot/util';
+
+import { useTranslation } from '../translate';
+import { SortedTargets } from '../types';
+import { Section, Title } from './index';
+import SpinnerWrap from './SpinnerWrap';
+
+interface Props {
+  targets: SortedTargets;
+}
+
+function SummaryNominators ({ targets: { maxNominatorsCount,
+  minNominatorBond,
+  nominatorActiveCount,
+  nominatorElectingCount,
+  nominatorMinActiveThreshold } }: Props) {
+  const { t } = useTranslation();
+
+  return (
+    <Card withBottomMargin>
+      <Title>{t<string>('nominators')}</Title>
+      <SummaryBox>
+        <Section>
+          <CardSummary
+            help={t<string>('Total count of nominators.')}
+            label={t<string>('total')}
+          >
+            <SpinnerWrap check={maxNominatorsCount}>
+              {formatNumber(maxNominatorsCount?.toNumber())}
+            </SpinnerWrap>
+          </CardSummary>
+        </Section>
+        <Section>
+          <CardSummary
+            help={t<string>('Electing count of nominators.')}
+            label={t<string>('electing')}
```

Comment:
```suggestion
            help={t<string>('Number of electing nominators.')}
            label={t<string>('electing')}
```

Reply:
This value seems to be incorrect and should be `const.electionProviderMultiPhase.maxElectingNominators`, i.e. 22,500 for polkadot.

Reply:
> This value seems to be incorrect and should be `const.electionProviderMultiPhase.maxElectingNominators`, i.e. 22,500 for polkadot.

[fixed here](https://github.com/polkadot-js/apps/pull/7179/commits/fc5ba1529067e318cb07840cbf8144bcbb7f80cc)